verified_model/Site/North/corsika_observation_level.json

```diff
@@ -0,0 +1,75 @@
+{
+    "value": 2156.0,
+    "units": "m",
+    "type": "numpy.float64",
+    "file": false,
+    "metadata": {
+        "CTA": {
+            "REFERENCE": {
+                "VERSION": "1.0.0"
+            },
+            "CONTACT": {
+                "ORGANIZATION": "CTAO",
+                "NAME": "CTAO Simulation Team",
+                "EMAIL": null
+            },
+            "PRODUCT": {
+                "DESCRIPTION": "Corsika observation level as used in prod6",
+                "CREATION_TIME": "2024-01-11T08:20:01",
```

Comment:
Just curious, does it say anywhere in the schema the format used here for the creation time. I think astropy as the option of "guess" anyway, but I wonder.

Reply:
Looked into the data model document again and don't see a rule. The examples use `2018-11-10 15:29:00` (but all examples disappeared in the newest version). I have used above `datetime.datetime.now().isoformat(timespec="seconds")`, which I think is reasonable and should be accepted by astropy. But I don't have a strong opinion.

Reply:
No strong opinion here but from short experience elsewhere, this specific format is better than the old example because it's easier to read in a table or file name since it doesn't have spaces. Just something to keep in mind for the future.